_posts/2024-05-25-unstable-release-0.10.7.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Unstable release 0.10.7
+author: Saugia
+---
+
+Hello! I heard you like updates. Well, you're in luck, because the newest version of Endless Sky, v0.10.7, is now live on both Steam and Github! This is an unstable release that brings with it a little bit of everything - new content, mechanics, visuals, balance changes, and so on. If you want to know exactly what that everything is, you can find the changelog for all the new changes in the update available here: https://github.com/endless-sky/endless-sky/blob/master/changelog
+
+The following are some highlights of notable changes and additions coming with this release.
+
+Have you ever wondered why cloaking paints your ship dark red, and all of a sudden nobody in the galaxy can see it? Well, thanks to Koranir, not only does your ship turn red, but a fancier outline for cloaking effects makes it partially transparent! This can be toggled within the settings.
+
+<img class="centered shadowed" src="/images/blog/v0.10.7/cloak.png" width="400" height="225" />
+
+While we're talking about new settings, Koranir has also introduced a new "Dynamic Camera Acceleration" setting that makes flying feel a bit more dynamic, with the camera panning away from your ship based on its acceleration. This is currently defaulted to on, but it can also be turned off in the settings. We may change it to default to off based on feedback.
+
+A neat new mechanic turrets can now be given is a "firing arc", where you can define the start and end angles that a turret hardpoint can rotate, as well as what direction it faces. This can be useful for if you want to make turrets that don't shoot through large structures on a ship model, or to give a ship blindspots that it can't fire turrets from 1010todd and oo13.
+
+On the content side, missions can now "mark" systems of importance without the player being required to visit them, as well as leave them marked after being visited. Not only that, but mission triggers and conversations can do various new things such as play music, give the player debt, and perform events immediately instead of having to wait at least one day. @Amazinite
+
+Around human space, if you keep your eyes peeled, you will find a few more planets and stations that weren't there last time, bringing with them a little of fluff. Some of these can be used as stopovers for content that otherwise didn't have a nearby planet to land at. Thanks to @DoomKorath and @bene_dictator for contributing these.
```

Comment:
```suggestion
Around human space, if you keep your eyes peeled, you will find a few more planets and stations that weren't there last time, bringing with them a little of fluff. Some of these can be used as stopovers for content that otherwise didn't have a nearby planet to land at. Thanks to DoomKorath and bene-dictator for contributing these.
```